src/database/DataTypes/Requisition.js

```diff
@@ -373,6 +375,35 @@ export class Requisition extends Realm.Object {
     database.delete('RequisitionItem', itemsToPrune);
   }
 
+  get canFinaliseRequest() {
```

Comment:
all of the `get canFinalise..` methods are ripped straight from the `finalisation.js` functions